test/ResponseTest.php

```diff
@@ -88,9 +88,6 @@ public function testBody(): void
         $response->setBody("foobar");
         $this->assertSame('foobar', $response->getBody()->buffer());
 
-        $response->setBody($response->getBody());
```

Comment:
Test was throwing an error that you can't `->buffer()` twice, which seems to be exactly what's being tested here - guessing that behaviour was removed.

Reply:
Yes, that has been removed in https://github.com/amphp/byte-stream/commit/2c3f3383bc7587ae1997ad3da40486e83351ff0d#diff-ebb3b05d8384827669dfc2fadbce197f021879aed9b438e97e931bb498e0cef2. The issues I see with offering it again is different cancellations / limits on multiple calls.

@trowski This has been a hidden change, but I think it was the correct thing to do, no?